usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -98,17 +100,36 @@ def get_download_response(self, file_name):
         else:
             file_path = self.s3_handler.get_simple_url(file_name=file_name)
 
-        # Add additional response elements that should be part of anything calling this function
-        response = {
-            "status": download_job.job_status.name,
-            "url": file_path,
-            "message": download_job.error_message,
-            "file_name": file_name,
-            # converting size from bytes to kilobytes if file_size isn't None
-            "total_size": download_job.file_size / 1000 if download_job.file_size else None,
-            "total_columns": download_job.number_of_columns,
-            "total_rows": download_job.number_of_rows,
-            "seconds_elapsed": download_job.seconds_elapsed(),
-        }
+        base_response = {"file_name": file_name, "file_url": file_path}
+        if is_download_status:
```

Comment:
What was the impetus for this boolean? Just so that the two endpoints can share the functionality of `base_response` and the boto code? It seems to me like this is two distinct views smashed into one. Splitting them up and keeping them in the same class would let you use all the common constants and functions you want and lower Cognitive Complexity.

Reply:
Good call, will make the change